master_key_rotation_for_disconnected_minions.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: Master Key Rotation for disconnected Minions
+- Start Date: 2021-05-27
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+There are several reasons to rotate keys:
+- Exposure or a compromised key
+- Security departments require key rotation on a periodic basis
+- It is common praxis to expire or revoke keys or certificates (e.g. [eset protect 1](https://help.eset.com/protect_admin/80/en-US/certificate_replacement.html), [eset protect 2](https://support.eset.com/en/kb6824-peer-certificate-is-going-to-expire-error-in-eset-security-management-center-7x))
+
+
+# Motivation
+[motivation]: #motivation
+
+If a minion is offline when the master changes its key, there is no way to get it after the fact. The minion will not be able to reconnect to the master after the master has changed its key. This is a problem e.g. with end user desktops/laptops that aren't ALWAYS connected to the master.
+
+The goal is a key management system that allows for:
+- Expiring and future (post-expire) keys
```

Comment:
Expiring and ~~expired~~ replacement keys

Reply:
Thank you for suggesting `replacement key` - this improves readability.